app/models/manageiq/providers/openstack/cloud_manager/vm.rb

```diff
@@ -163,6 +163,28 @@ def memory_mb_available?
     true
   end
 
+  def params_for_create_snapshot
+    {
+      :fields => [
+        {
+          :component  => 'text-field',
+          :name       => 'name',
+          :id         => 'name',
+          :label      => _('Name'),
+          :isRequired => true,
+          :validate   => [{:type => 'required'}],
+        },
+        {
+          :component => 'textarea',
+          :name      => 'description',
+          :id        => 'description',
+          :label     => _('Description'),
+          :validate  => [{:type => 'required'}],
```

Comment:
missing `isRequired` here?

Reply:
Fixing in https://github.com/ManageIQ/manageiq-providers-openstack/pull/672